socialsharing_diaspora/appinfo/info.xml

```diff
@@ -7,7 +7,9 @@
 	<description>
 		This allows the user to directly share link shares to diaspora
```

Comment:
This will look messed up as it will be considered a blockquote. I'll move that into one line everywhere here.